tests/test_timeseries.py

```diff
@@ -0,0 +1,48 @@
+from CovidTracker.get_covid_data import get_covid_data
+from CovidTracker.plot_time_series import plot_ts
+from pytest import raises
+import pandas as pd
+import matplotlib as plt
+import pytest
+
+
+def test_plot_ts():
```

Comment:
Should test more branches of the function. For example, if start and end dates are given, if given start date is earlier than min date in covid_df, etc